files/start.sh

```diff
@@ -73,6 +73,12 @@ fi
 # Change root password
 echo "root:${ROOT_PASSWORD}" | chpasswd
 
+# Change email address
+pushd /var/www/html/web
+drush sqlq "UPDATE users_field_data SET mail = 'admin@localhost' WHERE uid = 1"
```

Comment:
we can do this in the `drush site-install` using: 
--account-mail
--site-mail

Reply:
good idea, it works, done